pkg/client/observatorium/observability_config.go

```diff
@@ -22,9 +22,13 @@ const (
 
 type ObservabilityConfiguration struct {
 	// Red Hat SSO configuration
-	RedHatSsoAuthServerUrl     string `json:"redhat_sso_auth_server_url" yaml:"redhat_sso_auth_server_url"`
-	RedHatSsoRealm             string `json:"redhat_sso_realm" yaml:"redhat_sso_realm"`
-	RedHatSsoTenant            string `json:"redhat_sso_tenant" yaml:"redhat_sso_tenant"`
+	//// RedHatSsoTenant is used to specify the Observatorium tenant for
+	//// the Data Plane Remote Write configuration. It is not used for
+	//// the control plane connection against Observatorium
+	RedHatSsoTenant string `json:"redhat_sso_tenant" yaml:"redhat_sso_tenant"`
```

Comment:
I also added a commit to clarify the usage of the two remaining parameters.
As a future improvement I think refactoring this parameters might be a good idea: putting them into its own struct, renaming them or rethinking how we expose them, etc.

Reply:
+1 makes sense